tests/apply_rules.rs

```diff
@@ -117,8 +126,9 @@ fn test_ntw_consolidation() {
         "",
         "",
         "./tests/fixtures/apply_rules/ntw_consolidation.json",
-        "./tests/fixtures/apply_rules/output_consolidation",
+        "./tests/fixtures/apply_rules/output_ntw_consolidation",
```

Comment:
you can just keep these 2 files in folder (output_ntw_consolidation) and empty the rest

Reply:
and you can remove "output_consolidation" folder